ardupilot/equipment/power/20009.BatteryPeriodic.uavcan

```diff
@@ -0,0 +1,47 @@
+#
+# Battery data to be sent statically upon request or periodically at a low rate
+#
+
+# Should we have separate CAN fields for other types of battery measurement devices???? IE not smart_batteries but still on CAN? 
+# For example there are many CAN nodes that currently take an analog only monitor voltage only, current only, or current and voltage and sne d that data using the current smartbatt CAN message
+# I don't think having additional mesages for other types of monitors makes sense?
+#
+# uint32 measurement_capabilities   # type mask of capabilities?  measurement (voltage, cell voltages, temperatures, etc.)
+# uint32 estimation_capabilities    # estimator type ie columb counter, columb ++ impedance estimator, EKF 2 state, UKF, SOH estimated, others
+
+uint8[<=50] name                    # Formatted as manufacturer_product, 0 terminated
+uint8[<=16] serial_number           # Serial number in ASCII characters, 0 terminated
+uint8[<=11] manufacture_date        # Manufacture date (DD/MM/YYYY) in ASCII characters, 0 terminated
+uint32 design_capacity              # [mAh] Fully Charged Design Capacity. UINT32_MAX: field not provided.
+
+uint8 cells_in_series               # Number of battery cell in series. UINT8_MAX: field not provided.
+
+# Nominal voltage of the battery pack
+# the nominal Voltage can be used for conversion between W-h and A-h
+# if the value of this field is 0, the conversion should be omitted.
+float16 nominal_voltage             # [Volt]
+
+# Are these 5 fields below actually actionable? Do we really need these?
```

Comment:
These others are useful on systems that have a separate charger ie a solar panel, fuel cell or generator. 

Also they could be used to validate parameters on the the autopilot

For a few bytes it really gives the full specs. of the battery system. (Assuming this message can be requested by the system and isn't periodically sent.)

Though I just realized this doesn't include
`charging_current_maximum`   opps

Reply:
Fair enough. I added a `charging_maximum_current`. Also changed all voltage fields to float16 with units of volts